specs/README.md

```diff
@@ -0,0 +1,23 @@
+# VDK Enhancement Proposal (VEP)
+
+A VDK Enhancement Proposal (VEP) is a way to propose,
+communicate and coordinate on new efforts for the Versatile Data Kit project.
+
+## Goals
+
+- To help users and contributors understand what is being done and why is it being done this way
+  and reduce "tribal knowledge" in our community.
+- To help provide a better quality product by having peer review so that potential
+  problems and inconsistencies are caught early.
+- To help contributors develop their ideas into working solutions faster and easier
+  by having a common structure for proposing changes with important topics to cover.
+
+## Quick start for the VEP process:
+
+1. It is good idea to socialize an idea with some other contributors of VDK first. You can send your idea to VDK mailing list, slack, mail, etc.
+2. Follow the process outlined in the [VEP template](NNNN-template/README.md)
+
```

Comment:
are we considering a POC (interfaces, implementation of gap-solving snippets) might be a primary artifact to bootstrapping a VEP?

Reply:
That's upto developer.